project/src/views/new-evaluation/dropDownSelector.js

```diff
@@ -0,0 +1,25 @@
+const DropdownSelector = function(options = {}) {
+  
+  const arr = [`<legend class="field-title field-title-skin">${options.legends}</legend>`];
+  const constant = options.items.map((el) => {
+    const dropDownOptions= el.options.map((e) => `<option value="${e}">${e}</option>`)
+    const dropdownEl = dropDownOptions.join("");
+    const aux = el.label.split(' ').join('_');
+    
+    return `        
+            <ul>
+             <li>
+                <label class="firstLi" for="${aux}">${el.label}</label>
```

Comment:
Firstly, I wouldn't call `firstLi` `firstLi`. Secondly, I'd probably call `firstLi` something else... :stuck_out_tongue:

Reply:
:)) I'll remove it because I don't use it